src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -472,19 +520,27 @@ def _check_request(self, request: Request) -> Optional[Result]:
                 )
                 return Error(request=request, msg=msg)
 
-        if not request.adjust_power:
-            if request.power < 0:
-                bound = self._get_lower_bound(
-                    request.batteries, request.include_broken_batteries
-                )
-                if request.power < bound:
-                    return OutOfBound(request=request, bound=bound)
-            else:
-                bound = self._get_upper_bound(
-                    request.batteries, request.include_broken_batteries
-                )
-                if request.power > bound:
-                    return OutOfBound(request=request, bound=bound)
+        bounds = self._get_bounds(pairs_data)
+        if request.adjust_power:
+            # Automatic power adjustments can only bring down the requested power down
+            # to the inclusion bounds.
+            #
+            # If the requested power is in the exclusion bounds, it is NOT possible to
+            # increase it so that it is outside the exclusion bounds.
```

Comment:
This has been our practice I think.  Going towards zero can be automatic/not away from.  Increasing power has to be a considered decision, not automatic, because for example the minimum might be too high for sustained maintenance, and could lead to peaks or unexpected soc drops, etc.